test/e2e/scalability/density.go

```diff
@@ -209,7 +212,10 @@ func runDensityTest(dtc DensityTestConfig) time.Duration {
 	for i := range dtc.DaemonConfigs {
 		dtc.DaemonConfigs[i].Run()
 	}
+	auxObjectCreationPhase.End()
 
+	replicationCtrlStartupPhase := testPhaseDurations.StartPhase(300, "replication controller startup")
```

Comment:
Please change this msg (like "saturation pods creation" or sth). We're also using replication controllers for our density pods, you need to differentiate it from saturation pods (which btw don't need to come from RCs, they can also be deployments, RS, etc).

Reply:
Done.